libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -539,31 +539,31 @@ - (void) webView:(WKWebView *) webView decidePolicyForNavigationAction:(WKNaviga
         NSString *refreshUrl = [self isLoginRedirectUrl:navigationAction.request.URL];
         if (refreshUrl != nil) {
             [self log:SFLogLevelWarning msg:@"Caught login redirect from session timeout. Reauthenticating."];
-
+            
             /*
              * Reconfigure user agent. Basically this ensures that Cordova whitelisting won't apply to the
              * WKWebView that hosts the login screen (important for SSO outside of Salesforce domains).
              */
             [SFSDKWebUtils configureUserAgent:[self sfHybridViewUserAgentString]];
-            [[SFAuthenticationManager sharedManager]
-             loginWithCompletion:^(SFOAuthInfo *authInfo,SFUserAccount *userAccount) {
-                 [SFUserAccountManager sharedInstance].currentUser = userAccount;
-                 // Reset the user agent back to Cordova.
-                 [self authenticationCompletion:refreshUrl authInfo:authInfo];
-             } failure:^(SFOAuthInfo *authInfo, NSError *error) {
-                 if ([self logoutOnInvalidCredentials:error]) {
-                     [self log:SFLogLevelError msg:@"Could not refresh expired session. Logging out."];
-                     NSMutableDictionary *attributes = [[NSMutableDictionary alloc] init];
-                     attributes[@"errorCode"] = [NSNumber numberWithInteger:error.code];
-                     attributes[@"errorDescription"] = error.localizedDescription;
-                     [SFSDKEventBuilderHelper createAndStoreEvent:@"userLogout" userAccount:nil className:NSStringFromClass([self class]) attributes:attributes];
-                     [[SFAuthenticationManager sharedManager] logout];
-                 } else {
-                     
-                     // Error is not invalid credentials, or developer otherwise wants to handle it.
-                     [self loadErrorPageWithCode:error.code description:error.localizedDescription context:kErrorContextAuthExpiredSessionRefresh];
-                 }
-             } credentials:[SFUserAccountManager sharedInstance].currentUser.credentials];
+            [[SFAuthenticationManager sharedManager] refreshCredentials:[SFUserAccountManager sharedInstance].currentUser.credentials
```

Comment:
Looking at it, I think the old code dropped the arguments down a line, which gave them less whitespace.  I can fix that.